qiskit/aqua/algorithms/classifiers/qsvm/qsvm.py

```diff
@@ -207,7 +227,10 @@ def get_kernel_matrix(quantum_instance, feature_map, x1_vec, x2_vec=None):
             numpy.ndarray: 2-D matrix, N1xN2
         """
 
-        use_parameterized_circuits = feature_map.support_parameterized_circuit
+        if isinstance(feature_map, QuantumCircuit):
+            use_parameterized_circuits = True
+        else:
+            use_parameterized_circuits = feature_map.support_parameterized_circuit
```

Comment:
I thought it was, since `FeatureMap` types are still supported and they might not support parameterized circuits. But I cannot think of one that does not allow parameterized circuits.. @woodsp-ibm what do you think?

Reply:
summarizing Steve's and my discussion: yes, we can

Reply:
Edit: Not possible due to RawFeatureVector. Let's solve that after this release, should be straightforward enough but needs refactor of the StatevectorCircuit (this can be merged with the Initialize/Isometry logic)